README.md

```diff
@@ -205,6 +205,7 @@ terminate.
 | `arm-unknown-linux-gnueabihf`        | 2.27   | 7.3.0   | ✓   | 4.1.0 |   ✓    |
 | `arm-unknown-linux-musleabi`         | 1.1.20 | 6.3.0   |     | 4.1.0 |   ✓    |
 | `arm-unknown-linux-musleabihf`       | 1.1.20 | 6.3.0   |     | 4.1.0 |   ✓    |
+| `armv5te-unknown-linux-gnueabi`      | ???    | ???     | ??? | ???   |   ?    |
```

Comment:
is there a reason these are currently unspecified?

Reply:
Is there a straightforward way to print these versions in the container? I have a local one and could add these infos.